request: cloudflare worker support

### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

when bundling with cloudflare workers, I get an error:

```
    ../../node_modules/@sapphire/shapeshift/dist/index.mjs:2:24:
      2 │ import { inspect } from 'node:util';
        ╵                         ~~~~~~~~~~~

  The package "node:util" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, 
which will remove this error.


X [ERROR] Build failed with 1 error:

  ../../node_modules/@sapphire/shapeshift/dist/index.mjs:2:24: ERROR: Could not resolve "node:util"
```

### Desired solution

all that's needed is to remove the `node:` prefix, then the node_compat flag can actually work. Yeah it's really annoying.

### Alternatives considered

using another library I guess

### Additional context

_No response_

--- a/src/lib/errors/BaseError.ts
+++ b/src/lib/errors/BaseError.ts
@@ -1,4 +1,4 @@
-import type { InspectOptionsStylized } from 'node:util';
+import type { InspectOptionsStylized } from 'util';
 
 export const customInspectSymbol = Symbol.for('nodejs.util.inspect.custom');
 export const customInspectSymbolStackLess = Symbol.for('nodejs.util.inspect.custom.stack-less');

--- a/src/lib/errors/CombinedError.ts
+++ b/src/lib/errors/CombinedError.ts
@@ -1,4 +1,4 @@
-import type { InspectOptionsStylized } from 'node:util';
+import type { InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class CombinedError extends BaseError {

--- a/src/lib/errors/CombinedPropertyError.ts
+++ b/src/lib/errors/CombinedPropertyError.ts
@@ -1,4 +1,4 @@
-import type { InspectOptionsStylized } from 'node:util';
+import type { InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class CombinedPropertyError extends BaseError {

--- a/src/lib/errors/ExpectedConstraintError.ts
+++ b/src/lib/errors/ExpectedConstraintError.ts
@@ -1,4 +1,4 @@
-import { inspect, type InspectOptionsStylized } from 'node:util';
+import { inspect, type InspectOptionsStylized } from 'util';
 import { customInspectSymbolStackLess } from './BaseError';
 import { BaseConstraintError, type ConstraintErrorNames } from './BaseConstraintError';
 

--- a/src/lib/errors/ExpectedValidationError.ts
+++ b/src/lib/errors/ExpectedValidationError.ts
@@ -1,4 +1,4 @@
-import { inspect, type InspectOptionsStylized } from 'node:util';
+import { inspect, type InspectOptionsStylized } from 'util';
 import { customInspectSymbolStackLess } from './BaseError';
 import { ValidationError } from './ValidationError';
 

--- a/src/lib/errors/MissingPropertyError.ts
+++ b/src/lib/errors/MissingPropertyError.ts
@@ -1,4 +1,4 @@
-import type { InspectOptionsStylized } from 'node:util';
+import type { InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class MissingPropertyError extends BaseError {

--- a/src/lib/errors/MultiplePossibilitiesConstraintError.ts
+++ b/src/lib/errors/MultiplePossibilitiesConstraintError.ts
@@ -1,4 +1,4 @@
-import { inspect, type InspectOptionsStylized } from 'node:util';
+import { inspect, type InspectOptionsStylized } from 'util';
 import { customInspectSymbolStackLess } from './BaseError';
 import { BaseConstraintError, type ConstraintErrorNames } from './BaseConstraintError';
 

--- a/src/lib/errors/UnknownEnumValueError.ts
+++ b/src/lib/errors/UnknownEnumValueError.ts
@@ -1,4 +1,4 @@
-import type { InspectOptionsStylized } from 'node:util';
+import type { InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class UnknownEnumValueError extends BaseError {

--- a/src/lib/errors/UnknownPropertyError.ts
+++ b/src/lib/errors/UnknownPropertyError.ts
@@ -1,4 +1,4 @@
-import { inspect, type InspectOptionsStylized } from 'node:util';
+import { inspect, type InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class UnknownPropertyError extends BaseError {

--- a/src/lib/errors/ValidationError.ts
+++ b/src/lib/errors/ValidationError.ts
@@ -1,4 +1,4 @@
-import { inspect, type InspectOptionsStylized } from 'node:util';
+import { inspect, type InspectOptionsStylized } from 'util';
 import { BaseError, customInspectSymbolStackLess } from './BaseError';
 
 export class ValidationError extends BaseError {

